docs/DevGuide/CodeReviewGuide.md

```diff
@@ -15,73 +15,88 @@ Here we summarize what we view as the more important portions of the guides.
 Stylistic Items:
 
 * Adhere to [Google style](https://google.github.io/styleguide/cppguide.html).
-Can use
-[`clang-format -style=google` (here)](http://clang.llvm.org/docs/ClangFormat.html).
+  Can use [`clang-format -style=google` (here)]
+  (http://clang.llvm.org/docs/ClangFormat.html).
 * CamelCase: class names template parameters, file names, and directory names.
 * snake_case: function, variable, metafunction and metavariable names.
 * SCREAMING_SNAKE_CASE: macros.
-* Functions or classes only used internally in a library should be in a namespace named `LibraryOrFile_detail`. For example, `databox_detail`, `Tensor_detail` or `ConstantExpressions_detail`.
+* Functions or classes only used internally in a library should be in a
+  namespace named `LibraryOrFile_detail`. For example, `databox_detail`,
+  `Tensor_detail` or `ConstantExpressions_detail`.
 * Name unused function parameters `/*parameter_name*/` or `/*meta*/` for TMP
   cases
 * Type aliases that wrap type traits have a trailing `_t` following the STL
-* Private member variables have a [trailing underscore](https://google.github.io/styleguide/cppguide.html#Variable_Names).
-* Do not use [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation), e.g. `double* pd_blah` is bad
+* Private member variables have a [trailing underscore]
+  (https://google.github.io/styleguide/cppguide.html#Variable_Names).
```

Comment:
The link doesn't render correctly with the line break.  (At least in Github's renderer.)